### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by jonasspinner**
Here are some fixes for typos found by my IDE.

### CODE REVIEW

File: src/coord/localij.rs [hunk 1]
```diff
@@ -82,7 +82,7 @@ impl TryFrom<LocalIJK> for CellIndex {
             base_cell.map(BaseCell::is_pentagon).unwrap_or_default();
 
         if dir != Direction::Center {
-            // If the index is in a warped direction, we need to unwarp the base
+            // If the index is in a warped direction, we need to unwrap the base
```
Comment (grim7reaper) [inline src/coord/localij.rs:85]: this one is actually correct: https://en.wiktionary.org/wiki/unwarp

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Possible sections are:
 - fix link to CHANGELOG in CONTRIBUTING
 - fix spelling in bbox
 - escape backslash in README
+- fix typos
```
Comment (grim7reaper) [inline CHANGELOG.md:22]: I think we can rename the entry `fix spelling in bbox` to `fix various typos` to avoid too many similar changelog entries.

**Review by grim7reaper — CHANGES_REQUESTED** [review]
So many typos 🫣
Thanks for taking the time to fix them!

File: src/coord/localij.rs (hunk 1 above)
Comment (jonasspinner) [reply src/coord/localij.rs:85]: Thanks for catching that!

File: CHANGELOG.md (hunk 2 above)
Comment (jonasspinner) [reply CHANGELOG.md]: solved!

**Pull request merged**